Makefile

```diff
@@ -0,0 +1,13 @@
+build-toolchain:
+	GITHUB_ACTIONS=false CARGO_TARGET_RISCV32I_JOLT_ZKVM_ELF_RUSTFLAGS="-Cpasses=loweratomic" ./x build
+	GITHUB_ACTIONS=false CARGO_TARGET_RISCV32I_JOLT_ZKVM_ELF_RUSTFLAGS="-Cpasses=loweratomic" ./x build --stage 2
```

Comment:
what's this `--stage 2` thing about?

Reply:
Building the compiler happens in two stages. Tbh idk why but thats just how you build it.